docs/architecture/adr-047-extend-upgrade-plan.md

```diff
@@ -3,28 +3,28 @@
 ## Changelog
 
 * Nov, 23, 2021: Initial Draft
+* May 15, 2023: Remove `pre_run` and `post_run` fields for simplifying the ADR
 
 ## Status
 
-PROPOSED Not Implemented
+ACCEPTED Not Implemented
 
 ## Abstract
 
-This ADR expands the existing x/upgrade `Plan` proto message to include new fields for defining pre-run and post-run processes within upgrade tooling.
-It also defines a structure for providing downloadable artifacts involved in an upgrade.
+This ADR expands the existing x/upgrade `Plan` proto message to include new fields for defining a structure for providing downloadable artifacts involved in an upgrade.
 
 ## Context
 
 The `upgrade` module in conjunction with Cosmovisor are designed to facilitate and automate a blockchain's transition from one version to another.
 
 Users submit a software upgrade governance proposal containing an upgrade `Plan`.
-The [Plan](https://github.com/cosmos/cosmos-sdk/blob/v0.44.5/proto/cosmos/upgrade/v1beta1/upgrade.proto#L12) currently contains the following fields:
+The [Plan](https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/upgrade/v1beta1/upgrade.proto#L13-L45) currently contains the following fields:
```

Comment:
This link needs to be to a specific version instead of a link to whatever `main` currently has, specially since it include line numbers.